code/modules/mob/living/carbon/human/human.dm

```diff
@@ -596,6 +599,7 @@
 	else if(!(flags & SHOCK_NOGLOVES)) //This gets the siemens_coeff for all non tesla shocks
 		if(gloves)
 			siemens_coeff *= gloves.siemens_coefficient
+	siemens_coeff *= physiology.siemens_coeff
 	siemens_coeff *= dna.species.siemens_coeff
```

Comment:
Is this still meant to multiple by both?

Reply:
one is based on species, the other is based on physiology.